dpdata/plugins/abacus.py

```diff
@@ -1,7 +1,47 @@
+from ast import For
 import dpdata.abacus.scf
 import dpdata.abacus.md
 from dpdata.format import Format
 
+@Format.register("abacus/stru")
+@Format.register("stru")
+class AbacusSTRUFormat(Format):
+    def from_system(self, file_name, **kwargs):
+        return dpdata.abacus.scf.get_frame_from_stru(file_name)
+    def to_system(self, data, file_name, frame_idx=0, **kwargs):
+        """
+        Dump the system into ABACUS STRU format file.
+
+        Parameters
+        ----------
+        file_name : str
+            The output file name
+        frame_idx : int
+            The index of the frame to dump
+        pp_file: list of string, optional
+            List of pseudo potential files
+        numerical_orbital: list of string, optional
+            List of orbital files
+        mass: list of float, optional
+            List of atomic masses
+        numerical_descriptor: str, optional
+            numerical descriptor file
+        """
+        pp_file = None
+        numerical_orbital = None
+        mass = None
+        numerical_descriptor = None
+        if 'pp_file' in kwargs:
+            pp_file = kwargs['pp_file']
```

Comment:
can be written as `pp_file = kwargs.get('pp_file')`

Reply:
I have changed all of the cases into `kwargs.get()`.

Reply:
When you write as this case, line 34 can be removed.

Reply:
@Liu-RX plz remove the `if` clauses

Reply:
I have removed the `if` clauses and the `pp_file = None` lines.